src/guides/v2.2/cloud/architecture/scaled-architecture.md

```diff
@@ -0,0 +1,36 @@
+---
+group: cloud-guide
+title: Scaled architecture
+functional_areas:
+  - Cloud
+---
+
+The Cloud infrastructure scales according to your resource needs to achieve greater efficiency. The {{site.data.var.ece}} monitors your applications and automatically adjusts capacity to maintain steady, predictable performance. Converting to this architecture helps to mitigate problems, such as latency, large spikes in traffic, or running out of storage.
+
+## Split architecture
+
+Historically, the Pro architecture consisted of 3 nodes, each containing a full tech stack. Now, the scalable infrastructure provides a tiered solution with a minimum of 6 nodes: 3 nodes for the web server and 3 nodes for the database and other services. This split architecture provides the capability to scale tiers independently to achieve an optimal balance of performance. The database tier scales vertically (increases in size), and the web tier scales horizontally (increases instance count) and vertically (changes instance type and size).
+
+### Database tier scaling
```

Comment:
May will be better to use the name `Core nodes` / `Core tier` because these nodes contain not only DB, they contain ElasticSearch, Redis, DB, etc

@YPyltiai What do you think?

Reply:
Makes sense. Let's do that